src/main/domain/status/Status.js

```diff
@@ -49,6 +49,7 @@ export class Status {
   /**
    *
    * @param {Object} param
+   * @param {DomainEventBus} param.domainEventBus
```

Comment:
Why do you add this comment here if DomainEventBus is not a param in this class?

Reply:
ups, I left it here from a removed probe, removing it thanks!